src/components/ItemList.js

```diff
@@ -45,6 +51,17 @@ const ItemList = () => {
     setFilteredResults('');
   };
 
+  const handleDelete = async (id, name) => {
+    console.log(name);
+    const confirmation = window.confirm(
+      `Are you sure you want to delete ${name}?`,
```

Comment:
I appreciate that the confirmation popup box specifies the item you are deleting. Nice touch.

Reply:
Agree! That is slick.